src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -153,6 +156,23 @@ export default observer(function GamePage(): JSX.Element | null {
     gameInstallInfo?.manifest?.download_size || 0
   )
 
+  const hasRun = useRef(false)
+  useEffect(() => {
+    if (!action || hasRun.current) return
+    hasRun.current = true
```

Comment:
Do we need the `hasRun` var? Looks like an anti-pattern

Reply:
Yes, we need this otherwise the game launches twice.
This is just to make for the `useEffect` do not run twice.